v2/dtos/responses/event_test.go

```diff
@@ -80,27 +56,3 @@ func TestNewMultiEventsResponse(t *testing.T) {
 	assert.Equal(t, expectedMessage, actual.Message)
 	assert.Equal(t, expectedEvents, actual.Events)
 }
-
-func TestNewUpdateEventPushedByChecksumResponse(t *testing.T) {
-	expectedRequestId := "123456"
-	expectedStatusCode := 200
-	expectedMessage := "unit test message"
-	expectedChecksum := "04698a6f20feecb8bbf7cd01e59d31ba1ce17b24ba14b71a8fb370065d951f57"
-	actual := NewUpdateEventPushedByChecksumResponse(expectedRequestId, expectedMessage, expectedStatusCode, expectedChecksum)
-
-	assert.Equal(t, expectedRequestId, actual.RequestId)
-	assert.Equal(t, expectedStatusCode, actual.StatusCode)
-	assert.Equal(t, expectedMessage, actual.Message)
-	assert.Equal(t, expectedChecksum, actual.Checksum)
-}
```

Comment:
This isn't a `NoMessageResponse` test. Why removing?

I do think it will go way for another reason, which should be in a separate PR. That reason is with the new ID required and set from start we will no longer need checksum. So the EventPushedByChecksum API should also go away.

Reply:
it's a mistake.  I didn't have intention to remove this...
added it back.  thanks for the catching